pegasus/sites.v3/code.org/public/ai/index.haml

```diff
@@ -67,101 +65,92 @@ theme: responsive_full_width
-      =hoc_s("ai_curriculum_offerings.button")
+    %h2=hoc_s("ai_page.curricula.heading_middle_high")
+    = view :"ai_landing_page/ai_carousel_curricula_middle_high"
 
 %section.bg-neutral-light
   .wrapper
     %h2=hoc_s(:ai_carousel_heading)
     .quotes-wrapper
-      = view :ai_carousel_quotes
+      = view :"ai_landing_page/ai_carousel_quotes"
 
 %section#ai-pl
   .wrapper
     %h2=hoc_s(:ai_pl_heading)
-    .action-block__wrapper.action-block__wrapper--three-col
-      .action-block.action-block--one-col.flex-space-between
-        .content-wrapper
-          %p.overline
-            =hoc_s(:module_label_self_paced)
-          %h3=hoc_s(:ai_pl_101_title)
-          %img{src: "/images/self-paced-pl-tile-ai-101.jpg", alt: ""}
-          %p=hoc_s(:ai_pl_101_desc)
-        .content-footer
-          %p
-            %strong
-              =hoc_s(:module_label_duration)
-            =hoc_s(:module_duration_5_hrs)
-          %a.link-button{href: CDO.studio_url("/courses/self-paced-pl-ai-101")}
-            =hoc_s(:call_to_action_start_module)
-      .action-block.action-block--one-col.flex-space-between
-        .content-wrapper
-          %p.overline
-            =hoc_s(:module_label_video_series)
-          %h3=hoc_s(:ai_pl_101_hero_heading)
-          %img{src: "/shared/images/announcement/announcement_ai_pl_2023.png", alt: ""}
-          %p=hoc_s(:ai_pl_101_hero_desc)
-        .content-footer
-          %p
-            %strong
-              =hoc_s(:module_label_duration)
-            =hoc_s(:module_duration_5_hrs)
-          %a.link-button{href: "/ai/pl/101"}
-            =hoc_s(:call_to_action_watch_videos)
-      .action-block.action-block--one-col.flex-space-between
-        .content-wrapper
-          %p.overline
-            =hoc_s(:module_label_self_paced)
-          %h3=hoc_s(:ai_pl_title)
-          %img{src: "/images/ai/ai-professional-learning.png", alt: ""}
-          %p=hoc_s(:ai_pl_desc)
-        .content-footer
-          %p
-            %strong
-              =hoc_s(:module_label_duration)
-            =hoc_s(:module_duration_2_hrs)
-          %a.link-button{href: CDO.studio_url("/courses/self-paced-pl-aiml")}
-            =hoc_s(:call_to_action_start_module)
-
--# TODO - Remove the DCDO logic after the AI Teaching Assistant launch
-- if !!DCDO.get('ai-teaching-assistant-launch', false)
-  = view :section_divider_line
-
-  %section#ai-integrations
-    .wrapper
-      %h2=hoc_s("ai_integrations.heading")
-      %p=hoc_s("ai_integrations.desc")
-      = view :ai_integrations
-
-%section#ai-videos.bg-neutral-light
+    %p=hoc_s("ai_page.pl.desc")
+    -# TODO - Remove this DCDO flag logic once the 2024 curriculum is live
+    -# and remove everything below the else statement
+    - if !!DCDO.get('curriculum-launch-2024', false)
+      = view :"ai_landing_page/ai_carousel_pl"
+    - else
+      .action-block__wrapper.action-block__wrapper--three-col
```

Comment:
Nothing between this line and 140 changed, just weird GitHub diff formatting!

Reply:
As a tip, if you tell github to ignore whitespace in diffs, it doesn't show a diff! I often toggle that on PRs where things have shifted

Reply:
Whoaaa it hid it all 🙀 Ty for the tip!